tools/integration_tests/util/operations/file_operations.go

```diff
@@ -192,3 +194,38 @@ func ReadFileSequentially(filePath string, chunkSize int64) (content []byte, err
 	}
 	return
 }
+
+func WriteFileSequentially(filePath string, fileSize int64, chunkSize int64) (err error) {
+	var numberOfBytes int
+	var offset int64 = 0
+
+	file, err := os.OpenFile(filePath, os.O_RDWR|syscall.O_DIRECT|os.O_CREATE, FilePermission_0600)
+	if err != nil {
+		log.Printf("Error in opening file:%v", err)
+	}
+
+	// Closing file at the end.
+	defer CloseFile(file)
+
+	for offset < fileSize {
+		// Get random 200MB or remaining filesize data into chunk.
+		chunkSize = int64(math.Min(float64(chunkSize), float64(fileSize-offset)))
```

Comment:
Why are we converting back and forth (int to float & float to int)?
Instead:
if ((fileSize - offset) < chunkSize) {
    chunkSize = (fileSize - offset);
}